ui/src/holdingpen-new/components/Links.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Map } from 'immutable';
+import {
+  LinkOutlined,
+  LinkedinOutlined,
+  TwitterOutlined,
+} from '@ant-design/icons';
+
+import orcidLogo from '../../common/assets/orcid.svg';
+
+interface LinksProps {
+  urls: Map<string, any>;
+  ids: Map<string, any>;
+}
+
+const Links: React.FC<LinksProps> = ({ urls, ids }) => {
```

Comment:
can we use the existing one from hep?

Reply:
Similar components from hep are connected to Matomo tracking, I would have to either rewrite all of them to conditionally use matomo wrappers (ugly) or we will track actions from Holdingpen in hep's Matomo. Plus, they would need to conditionally change display, as in hep they look different (as below). I'd rather use the new component. 
<img width="327" alt="Screenshot 2024-08-12 at 09 38 08" src="https://github.com/user-attachments/assets/1df5e7e5-f114-45e9-b8fc-76830b225d37">